pkg/operators/klusterlet/controllers/klusterletcontroller/klusterlet_controller.go

```diff
@@ -712,20 +712,6 @@ func (n *klusterletController) cleanUp(
 		}
 	}
 
-	// remove the CRDs
```

Comment:
do we need to remove the CR?

Reply:
As we discussed in slack, it's fine to leave the CRs for now.

Reply:
please add a comment here indicating we do not delete crd and CR and the reason.